packages/travix-ui-kit/components/stepper/stepper.md

```diff
@@ -0,0 +1,35 @@
+Basic input:
+
+    <div style={{ background: '#D5E8F8', padding: '10px' }}>
+      <style dangerouslySetInnerHTML={{__html: `
+        .ui-stepper .ui-stepper-button.ui-stepper-button_decrease:before {
```

Comment:
Why it's not in the component style file?

Reply:
We need it only for demo in ui-kit. In RWD we are using correct icons